Test/CreateFile.Tests.ps1

```diff
@@ -0,0 +1,309 @@
+    }
+}
+
+Describe -Name 'Createfile.when the path already exists' {
+    It 'should only create the first file' {
+        Init
+        GivenWhiskeyYml @'
+        Build:
+        - CreateFile:
+            Path: file.txt
+            Content: "First File."
+
+        - CreateFile:
+            Path: file.txt
+            Content: "Second File... First File Overwritten"
+            Force: N
+'@
+        WhenRunningTask -ErrorAction SilentlyContinue
+        ThenTaskFailed
+        ThenFile -Path 'file.txt' -Contains 'First File.'
+        ThenError -Matches "'Path' already exists. Please change 'path' to create new file."
+    }
+}
+
+Describe -Name 'Createfile.when the path already exists and Force enabled' {
+    It 'should only create the first file' {
+        Init
+        GivenWhiskeyYml @'
+        Build:
+        - CreateFile:
+            Path: file.txt
+            Content: "First File."
+        
+        - CreateFile:
+            Path: file.txt
+            Content: "Second File... First File Overwritten"
+            Force: true
+'@
+        WhenRunningTask -ErrorAction SilentlyContinue
+        ThenSuccess
+        ThenFile -Path 'file.txt' -Contains "Second File... First File Overwritten"
+    }
+}
+
+Describe -Name 'Createfile.when the path is outside root' {
+    It 'should not create the file' {
+        Init
+        GivenWhiskeyYml @'
+        Build:
+        - CreateFile:
+            Path: ../file.txt
+            Content: "File above root."
+'@
+        WhenRunningTask -ErrorAction SilentlyContinue
+        ThenTaskFailed
+        ThenFile -Path '../file.txt' -Contains "File above root."
+    }
+}
+
+Describe -Name 'Createfile.when the path does not contain a file extension' {
+    It 'should create the file' {
+        Init
+        GivenWhiskeyYml @'
+        Build:
+        - CreateFile:
+            Path: file
+'@
+        WhenRunningTask -ErrorAction SilentlyContinue
+        ThenSuccess
+        ThenFile -Path 'file'
+    }
+}
+
+Describe -Name 'CreateFile.when a directory already exists with same path name' {
+    It -Name 'should not create the file'{
+        Init
+        GivenDirectory -Path 'file.txt'
+        GivenWhiskeyYml @'
+        Build:
+        - CreateFile:
+            Path: file.txt
+'@
+        WhenRunningTask -ErrorAction SilentlyContinue
+        ThenTaskFailed
+        ThenError -Matches "'Path' already points to a directory of the same name."
+    }
+ }
+
+ Describe -Name 'CreateFile.when a directory already exists with same path name and Force enabled' {
+    It -Name 'should not create the file'{
+        Init
+        GivenDirectory -Path 'file.txt'
+        GivenWhiskeyYml @'
+        Build:
+        - CreateFile:
+            Path: file.txt
+            Force: True
+'@
+        WhenRunningTask -ErrorAction SilentlyContinue
+        ThenTaskFailed
+        ThenError -Matches "'Path' already points to a directory of the same name."

```

Comment:
Change error message to `Unable to create file "{0}": a directory exists at that path.` Where `{0}` is the path to the directory.